src/app/views.py

```diff
@@ -171,6 +173,76 @@ def validate(request):
     else :
         return HttpResponseRedirect(settings.LOGIN_URL)
 
+def validate_xml(request):
+    """ View to validate xml text, used in the editor """
+    if request.user.is_authenticated() or settings.ANONYMOUS_LOGIN_ENABLED:
+        context_dict={}
+        if request.method == 'POST':
+            ajaxdict=dict()
+            try :
+                if "xmlText" in request.POST:
+                    """ Saving file to the media directory """
+                    schema_url = settings.BASE_DIR + "/examples/xml-schema.xsd"
+                    xmlText = request.POST['xmlText']
```

Comment:
Note that the XSD copy used here could get stale.  For the Java tools I pulled the XSD from the git repository source and if that failed, I used a local copy (see [LicenseXmlDocument.getSchema()](https://github.com/spdx/LicenseListPublisher/blob/02ea7d9ef77dc9a2b8cfc262dac52d6bed76b21b/src/org/spdx/licensexml/LicenseXmlDocument.java#L94)